### PR TITLE
Adding eval_data_viz to Explanation

### PR DESCRIPTION
**Pull request by marksoper**
eval_data is an input to explain_xxx(), and has been kept on the Explanation object in the 'eval_data' property.

This field would not be included in a serialization, upload/download of the Explanation.

To support consistent visualization of Explanations, we want to maintain a 'eval_data_viz' property on the Explanation that IS suitable for serialization and upload/download.

We establish a limit on the size of (configurable) 5K rows (an amount suitable for viz) so when eval_data is > 5K it will be randomly sampled down to 5K.

### CODE REVIEW

File: python/interpret_community/common/constants.py [hunk 1]
```diff
@@ -251,3 +253,6 @@ class SHAPDefaults(object):
     """Provide constants for default values to shap."""
 
     INDEPENDENT = 'independent'
+
```
Comment (imatiach-msft) [inline python/interpret_community/common/constants.py]: remove extra whitespace (?)

File: visualization/dashboard/src/Localization/en-with-comments.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
+{
+    // Prompts the user to select a point
```
Comment (imatiach-msft) [inline visualization/dashboard/src/Localization/en-with-comments.json]: why is this modified?  looks like there was a merge conflict?

**Comment by marksoper** [comment]
merge error, abandoning
